tasks/stats.js

```diff
@@ -123,7 +123,7 @@ function getMainBundleInfo() {
         '> The minified version of each partial bundle is also published to npm in a separate "dist-min" package.',
         '',
         [
-            '> The strict partial bundle includes everything except the traces that require function constructors.',
+            '> The strict partial bundle now includes all traces, noting that for regl-based traces the functions are provided inside the bundle.',
```

Comment:
```suggestion
            '> The strict bundle now includes all traces, but the regl-based traces are built differently to avoid function constructors.',
            '> This results in about a 10% larger bundle size, which is why we don't use this method by default.',
```
Please also update the line below to something like:
`Over time we intend to use the strict bundle to work on other strict CSP issues`